unlock-app/src/hooks/useKeyManager.ts

```diff
@@ -13,6 +13,7 @@ interface Options {
   enabled?: boolean
 }
 
+// todo: remove? not used anywhere
```

Comment:
@searchableguy wdyt? was introduces last month by you is that needed?

Reply:
Yes. I wanted to to use it for stimulating.